packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -92,10 +92,10 @@ class RibbonPluginImpl implements RibbonPlugin {
         key: T,
         strings?: LocalizedStrings<T>
     ) {
-        if (this.editor && this.uiUtilities) {
+        if (this.editor && this.uiUtilities && this.formatState) {
```

Comment:
If this.formatState is null (which is possible in some cases), it will cause button click not work. We should not add this parameter.

If a button needs to know current format state, it can call getFormatState(editor), plugin doesn't need to provide it.

Reply:
button onClick should still be called even formatState is null

Reply:
Here is my repro
![buttonbug](https://user-images.githubusercontent.com/23065085/217969018-dc7c92aa-19c2-4197-bd19-8f871756de50.gif)

Reply:
Here is my repro
![buttonbug](https://user-images.githubusercontent.com/23065085/217969018-dc7c92aa-19c2-4197-bd19-8f871756de50.gif)